CODING_STYLE.md

```diff
@@ -55,6 +55,7 @@ To set indentation and tab width settings uniformly, the repository contains an
 3. All symbols should be declared in a namespace except for final applications.
 4. Use anonymous namespaces for helpers whose scope is a cpp file only.
 5. Preprocessor symbols should be prefixed with the namespace in all-caps and an underscore.
+6. Do not use `std::` qualifier in cpp files (see 2.), except for `std::move`, which will otherwise cause the `check_style` step to fail.
```

Comment:
```suggestion
6. Do not use `std::` qualifier in cpp files (see 2.), except for `std::move` (this is an exception to the rule, failing which  will otherwise cause the `check_style` step to fail ).
```

Reply:
I don't see this making it any clearer, just more verbose.

Reply:
I agree. The current wording is pretty clear.

We might add something about *why* `std::move` is an exception but IMO it's also fine without it.